### PR TITLE
[SGP-14663] make it possible to test with utf8mb4

### PR DESCRIPTION
**Pull request by rbevers**
Prevents clobbering of `TEST` settings for a database that include `{"CHARSET": "utf8mb4", "COLLATION": "utf8mb4_unicode_ci"}`.

### CODE REVIEW

File: mezzanine/__init__.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 
-__version__ = "4.3.1"
+__version__ = "4.3.2"
```
Comment (caffodian) [inline mezzanine/__init__.py]: this will make it really annoying to be able to contribute this PR upstream :/

**Review by caffodian — APPROVED** [review]

File: mezzanine/__init__.py (hunk 1 above)
Comment (rbevers) [reply mezzanine/__init__.py]: Had to do it to ensure `pip install` actually uses the newest one. Does it support a `4.3.1utf8mb4` syntax. I'll try it.

**Review by jacobhamblin — APPROVED** [review]
:shipit:

File: mezzanine/__init__.py [hunk 2]
```diff
@@ -1,2 +1,2 @@
 
-__version__ = "4.3.1"
+__version__ = "4.3.1.post1"
```
Comment (rbevers) [inline mezzanine/__init__.py:2]: Still PEP-440 happy.

Comment (caffodian) [reply mezzanine/__init__.py:2]: Cool
This is small enough that I guess spinning up a branch w/o this one change (and submitting just the below 3 lines as a PR) would be okay.

**Review by caffodian — APPROVED** [review]

**Review by jacobhamblin — APPROVED** [review]
:shipit:

**Pull request merged**